### PR TITLE
[DNS] Add tenant custom nameservers and update ACNS conditions

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-6847

### CODE REVIEW

**Comment by dev2dark** [comment]
### **oس ننتون ok** ##

File: content/dns/_partials/_acns-tcns-conditions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  {{<example>}}
+
+  | Type | Name | Content |
+  | --- | --- | --- |
+  | `A` | `ns1.example.com` | `<IPv4>` |
+
+  {{</example>}}
+
+2. $3 can create up to five different $1 custom nameserver sets. Each nameserver set must have between two and five different nameserver names (`ns_name`) and each name cannot belong to more than one set. For example, if `ns1.example.com` is part of `ns_set 1` it cannot be part of `ns_set 2` or vice versa.
+3. [Subdomain](/dns/zone-setups/subdomain-setup/) or [Reverse](/dns/additional-options/reverse-zones/) zones can use $1 custom nameservers as long as they use a different nameserver set (`ns_set`) than their parent, child, or any other zone in their direct hierarchy tree.
```
Comment (jason-cf) [inline content/dns/_partials/_acns-tcns-conditions.md:20]:
```suggestion
3. [Subdomain](/dns/zone-setups/subdomain-setup/) or [Reverse zones](/dns/additional-options/reverse-zones/)  can use $1 custom nameservers as long as they use a different nameserver set (`ns_set`) than their parent, child, or any other zone in their direct hierarchy tree.
```

Comment (jason-cf) [inline content/dns/_partials/_acns-tcns-conditions.md]:
```suggestion
3. [Subdomain setup](/dns/zone-setups/subdomain-setup/) or [Reverse zones](/dns/additional-options/reverse-zones/) can use $1 custom nameservers as long as they use a different nameserver set (`ns_set`) than their parent, child, or any other zone in their direct hierarchy tree.
```

File: content/dns/additional-options/custom-nameservers/tenant-custom-nameservers.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+{{<render file="_ns-set-omission-callout.md">}}
+
+### Get a list of all TCNS names
+
+To get a list of all TCNS names in your tenant account, use the following API request:
+
+```bash
+$ curl https://api.cloudflare.com/client/v4/tenants/{tenant_id}/custom_ns \
+  -- header "X-Auth-Email: <EMAIL>" \
+  -- header "X-Auth-Key: <KEY>" \
```
Comment (jason-cf) [inline content/dns/additional-options/custom-nameservers/tenant-custom-nameservers.md]:
```suggestion
  --header "X-Auth-Email: <EMAIL>" \
  --header "X-Auth-Key: <KEY>"
```

**Review by jason-cf — APPROVED** [review]

**Pull request merged**